web/app/scripts/filterbar/options-directive.js

```diff
@@ -35,11 +38,22 @@
                  * @param filterLabel {string} label of which field to filter
                  */
                 scope.updateFilter = function(filterLabel) {
-                    // TODO: Make this directive create an array of vals to check for containment
-                    // instead of this annoying hack
-                    var filter = angular.copy(scope.filter);
-                    filter.contains = [filter.contains];
-                    filterbarController.updateFilter(filterLabel, _.merge({'_rule_type': 'containment'}, filter));
+                    // only include filters that actually do something
+                    if (scope.filter.contains.length) {
+                        // handle syntactic differences necessitated by having related objects
+                        if (scope.data.multiple) {
+                            // TODO: Implement a filter for related containment in djsonb
+                            filterbarController.updateFilter(filterLabel,
+                                                             _.merge({'_rule_type': 'related_containment'},
```

Comment:
I'm inclined to agree that the name I chose is not adequate - I also don't like `multiple_containment` because both support filtering for multiple values. What are your thoughts?